<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCNMainWindow</class>
 <widget class="QMainWindow" name="PCNMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC-NAVTEX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pc-navtex.qrc">
    <normaloff>:/resources/icon32x32/pc-navtex.png</normaloff>:/resources/icon32x32/pc-navtex.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="tbtA">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TAp.png</normaloff>
          <normalon>:/resources/icon32x32/TAa.png</normalon>:/resources/icon32x32/TAp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbtB">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TBp.png</normaloff>
          <normalon>:/resources/icon32x32/TBa.png</normalon>:/resources/icon32x32/TBp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbtC">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TCp.png</normaloff>
          <normalon>:/resources/icon32x32/TCa.png</normalon>:/resources/icon32x32/TCp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbtD">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TDp.png</normaloff>
          <normalon>:/resources/icon32x32/TDa.png</normalon>:/resources/icon32x32/TDp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbtE">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TEp.png</normaloff>
          <normalon>:/resources/icon32x32/TEa.png</normalon>:/resources/icon32x32/TEp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt0">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pc-navtex.qrc">
          <normaloff>:/resources/icon32x32/TZp.png</normaloff>
          <normalon>:/resources/icon32x32/TZa.png</normalon>:/resources/icon32x32/TZp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt00">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt01">
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt02">
        <property name="text">
         <string>02</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt03">
        <property name="text">
         <string>03</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt04">
        <property name="text">
         <string>04</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt05">
        <property name="text">
         <string>05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt06">
        <property name="text">
         <string>06</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt07">
        <property name="text">
         <string>07</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt08">
        <property name="text">
         <string>08</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt09">
        <property name="text">
         <string>09</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt10">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbt11">
        <property name="text">
         <string>11</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labMessage">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pc-navtex.qrc">:/resources/icon32x32/message-passiv.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labDownload">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pc-navtex.qrc">:/resources/icon32x32/down-passiv.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="tbvMessages">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="tedMessage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1660</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="actionStations_Filter_490"/>
    <addaction name="actionStations_Filter_518"/>
    <addaction name="actionStation_Names"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionCustomizeToolbar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionStations_Filter_518">
   <property name="text">
    <string>&amp;Stations Filter 518 kHz</string>
   </property>
  </action>
  <action name="actionStation_Names">
   <property name="text">
    <string>Station &amp;Names</string>
   </property>
  </action>
  <action name="actionStations_Filter_490">
   <property name="text">
    <string>Stations &amp;Filter 490 kHz</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionCustomizeToolbar">
   <property name="text">
    <string>Customize Toolbar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="pc-navtex.qrc">
     <normaloff>:/resources/icon24x24/help.png</normaloff>:/resources/icon24x24/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pc-navtex.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>PCNMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
